sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -726,6 +727,24 @@ def test_output_no_ansi(gallery_conf, script_vars):
     assert output_test_string.split('\n')[-1] == "0.25"
 
 
+def test_absl_logging(gallery_conf, script_vars):
+    """Test using absl logging does not throw error. This is important, as many
+    popular libraries like Tensorflow use absl for logging.
+    """
```

Comment:
It's easy enough to make the transition a tiny bit easier for other devs who might `git merge upstream/main` or so, so let's do it
```suggestion
    """
    pytest.importorskip('absl')
```